regcomp.c

```diff
@@ -19443,7 +19448,7 @@ S_optimize_regclass(pTHX_
 
```

Comment:
In the original PR @hvds said:
Is there any reason not to use the more obvious ternary here: *invert ? NPOSIXL : POSIXL? (Maybe for a future commit.)
@khwilliamson responded

Yes, if this is done, It shouldn't be in this commit, which is trying to avoid doing more than necessary in splitting the function.

Your proposal is clearer. A reason not to do what you suggest is because this is a repeat of a common paradigm in this file, so it isn't out of the blue, and it works without change if a new type of POSIXL regnode (and inverse) is added, which has long been a possibility and remains so.

====
And that got committed, but without real resolution